app/build.gradle

```diff
@@ -103,6 +102,7 @@ dependencies {
 
     compile 'com.squareup.sqlbrite:sqlbrite:0.5.0'
     compile "com.squareup.retrofit2:retrofit:$RETROFIT_VERSION"
+    compile "com.squareup.okhttp3:okhttp:3.4.1"
```

Comment:
I don't think we need this import as okhttp is not used in this boilerplate project anywhere.